mw-webapp/src/converter/currentProblemConverter.ts

```diff
@@ -0,0 +1,18 @@
+import {querySnapshot} from "src/converter/querySnapshot";
+import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
+import {CurrentProblem as CurrentProblemDTO} from "src/model/firebaseCollection/CurrentProblem";
+
+export const querySnapshotToCurrentProblemDTOConverter = (currentProblemsRaw: querySnapshot) => {
+  const currentProblemsDTO: CurrentProblemDTO[] = currentProblemsRaw.docs.map((currentProblemRaw) => ({
+    uuid: currentProblemRaw.data().uuid,
+    description: currentProblemRaw.data().description,
+    isDone: currentProblemRaw.data().isDone,
```

Comment:
callback in the map looks like converter itself

Reply:
why do you have DTO after converting? It is starnge a litle. DTO should be kind of schema for  json request responce

Reply:
because we get querySbapshot from firestore then we should convert it to DTO and then to buisness model

Reply:
why could not we convert something to somethign with only one converter?

Reply:
Done